packages/lib/services/plugins/api/JoplinClipboard.ts

```diff
@@ -59,4 +61,36 @@ export default class JoplinClipboard {
 		return this.electronClipboard_.availableFormats();
 	}
 
+	/**
+	 * Writes multiple formats to the clipboard simultaneously.
+	 * This allows setting both text/plain and text/html at the same time.
+	 *
+	 * <span class="platform-desktop">desktop</span>
+	 *
+	 * @example
+	 * ```typescript
+	 * await joplin.clipboard.writeMultiple({
+	 *   text: 'Plain text version',
+	 *   html: '<strong>HTML version</strong>'
+	 * });
+	 * ```
+	 */
+	public async writeMultiple(content: ClipboardContent): Promise<void> {
```

Comment:
That looks good, thank you! Since this is the generic method to write to clipboard, could it be called just `write()`?